src/apps/mpx.erl

```diff
@@ -112,7 +116,7 @@ handle_call({client_disconnected, #de_client{meta=Apps} = Client}, _, State) ->
 handle_call({handle_client_message, #de_client{data=AppData}=Client, App, Message}, _, State) ->
   Handler = deliverly_server:find_handler(App),
   AppClient = app_client(App, Client),
-  Reply = 
+  Reply =
     case Handler:handle_client_message(AppClient, de_client:decode(AppClient, Message)) of
```

Comment:
Line 120 is too long:     case Handler:handle_client_message(AppClient, de_client:decode(AppClient, Message)) of.